patches/@oclif+dev-cli+1.22.0.patch

```diff
@@ -64,14 +64,34 @@ index 3e613e0..4ed799c 100644
 +        ];
 +        for (const source of sources) {
 +            console.log(`cp "${source}" -> "${ws}"`);
-+            await qq.cp(path.join(c.root, source), ws);
++            try {
++                await qq.cp(path.join(c.root, source), ws);
++            } catch (err) {
++                // OK if package-lock.json doesn't exist
++                if (source !== 'package-lock.json') {
++                    throw err;
++                }
++            }
 +        }
 +        await qq.rm('bin/run.cmd');
 +    }
      const updatePJSON = async () => {
          qq.cd(c.workspace());
          const pjson = await qq.readJSON('package.json');
-@@ -124,7 +142,8 @@ async function build(c, options = {}) {
+@@ -55,7 +80,11 @@ async function build(c, options = {}) {
+             if (!await qq.exists(lockpath)) {
+                 lockpath = qq.join(c.root, 'npm-shrinkwrap.json');
+             }
+-            await qq.cp(lockpath, '.');
++            try {
++                await qq.cp(lockpath, '.');
++            } catch (err) {
++                console.log('WARNING: found neiter package-lock.json nor npm-shrinkwrap.json')
```

Comment:
`neiter` => `neither`

Reply:
Thanks! I'll fix this typo in another PR I'm working on. 👍